docs/user-guide/install.md

```diff
@@ -57,7 +94,11 @@ from this method of node lookup instead.
 Install with `sudo apt install asl3-update-nodelist`.
 
 ## Node Configuration
-Next step is to configure the node settings. YouTuber Freddie Mac has a nice ASL3 RPi installation and configuration video. See the part where the asl-menu is shown [https://youtu.be/aeuj-yI8qrU](https://youtu.be/aeuj-yI8qrU). Also see [ASL3 Menu](menu.md) for details.
+Next step is to configure the node settings. It is recommended to use the
+`asl-menu` command to set configuration for common use cases. See
```

Comment:
"to set the configuration" (add "the")

and a possible alternative to "set" might be "manage"/

Reply:
fixed